pkg/logging/logfields/logfields.go

```diff
@@ -63,6 +63,15 @@ const (
 	// IPAddr is an IPV4 or IPv6 address
 	IPAddr = "ipAddr"
 
+	// IPv4 is an IPv4 address
+	IPv4 = "ipv4"
+
+	// IPv6 is an IPv6 address
+	IPv6 = "ipv6"
+
+	// BuildTime is the time elapsed to build the BPF program of an endpoint
+	BuildTime = "buildTime"
```

Comment:
Could we call this bpfBuildTime to allow us to print other buildTimes later?

Reply:
Oh, also, `buildDuration` might be better. Time can mean the absolute timestamp in some contexts

Reply:
Is the idea to keep all fields unique to specific purpose? Won't the context in which it is printed provide that? the presence of an endpointID in the same log message means that the buildDuration is for the endpoint.

Reply:
I meant distinguishing between the time taken with BPF vs a policy resolution for the endpoint (and whatever else we want to track it the future). If we did print them in one summary line we couldn't use `build` again. I think we, internally, might always mean the bpf parts when we say build so I guess I should just learn convention.